src/main/resources/templates/admin.html

```diff
@@ -198,16 +199,15 @@ <h2 class="card-title mt-md-1 mt-0 mb-1">Manage Person</h2>
                                     </div>
                                 </div>
                             </form>
-                            <!--message for no groupings selected-->
-
-                            <div ng-show="!errorDismissed2 && !loading && emptySelect"
-                                 class="alert alert-danger form-alert fade show"
-                                 role="alert">
-                                <p th:text="#{screen.message.modal.search.noChecks}"></p>
-                            </div>
-
                         </div>
-
+                        <div class="break"></div>
```

Comment:
If  you intention was to make a break here I would not recommend using a div tag and adding css to it. The more conventional way to add a break is to use the br tag in HTML. If it was not to make a break then i would recommend renaming your css class as it can be confusing to members whether you are making a break or not.

Reply:
This is a break for pushing all the elements after it to the next line. I tried using <br> but it doesn't work with display:flex elements. I can rename the class to line-break to make it more clear.